### DISCUSSION TITLE
Unable to start NodeJS listening over HTTPS

### DISCUSSION THREAD

**Discussion by ipetrov-amatas** [outline/outline#1322] [General]
I've got the `.env` file with my front-facing domain name and port `3000`, but whenever I run `node index.js` it starts listening on `http://localhost:3000`.

Due to the fact that I've also got Apache acting as a reverse proxy with a redirect from `http://` to `https://`, I am unable to access `https://example.com:3000` so I can carry on with the Slack sign-in.

**Comment by wlatic** [comment]
Do you have a specific error for this? I know its been a long time, but an SSL error is spit out by postgres issues and it had me confused for an hour or more.

I've got openlitespeed running which allows index.js to run natively, and CloudFlare in front to deal with the SSL.
